qa/L0_parameters/parameters_test.py

```diff
@@ -148,7 +177,8 @@ def test_http_async_parameter(self):
         for parameters in parameter_list:
             result = self.http.async_infer(model_name='parameter',
                                            inputs=inputs,
-                                           parameters=parameters).get_result()
+                                           parameters=parameters,
+                                           headers=self.headers).get_result()
```

Comment:
Just realize that you are testing both parameter and header together, does it make sense to break into two test cases?
```
def test_xxx_parameter(self):
    ...
def test_xxx_parameter_in_header(self):
    ...
```

Reply:
I think it makes sense but we have 8 unit tests. I would have to create header variant for all of them. I think this results in more code duplication so I'd rather to keep the current structure.